lib/interceptor.js

```diff
@@ -78,9 +78,11 @@ Interceptor.prototype.replyWithError = function replyWithError(errorMessage) {
 }
 
 Interceptor.prototype.reply = function reply(statusCode, body, rawHeaders) {
-  if (arguments.length <= 2 && _.isFunction(statusCode)) {
+  // support the format of only passing in a callback
+  if (_.isFunction(statusCode)) {
     body = statusCode
     statusCode = 200
+    rawHeaders = undefined
```

Comment:
If we wanted to be more strict, instead of this we could throw an error if the first param is a function and `arguments.length > 1`.